doc/sphinx-guides/source/user/dataverse-management.rst

```diff
@@ -69,7 +69,8 @@ The Theme features provides you with a way to customize the look of your Dataver
 
```

Comment:
Can we please have a release note snippet?

Reply:
Yes, it's added with your suggestions, thanks ! I also add recommendation viewed at https://dataverse-guide--10445.org.readthedocs.build/en/10445/developers/version-control.html#writing-a-release-note-snippet

Reply:
Looks great! Thank you!